sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/OfflineStorageTests.cs

```diff
@@ -207,14 +212,14 @@ protected override IEnumerable<PersistentBlob> OnGetBlobs()
 
             protected override bool OnTryCreateBlob(byte[] buffer, int leasePeriodMilliseconds, out PersistentBlob blob)
             {
-                blob = new MockFileBlob();
+                blob = new MockFileBlob(_mockStorage);
                 this._mockStorage.Add(blob);
                 return blob.TryWrite(buffer);
             }
 
             protected override bool OnTryCreateBlob(byte[] buffer, out PersistentBlob blob)
             {
-                blob = new MockFileBlob();
+                blob = new MockFileBlob(_mockStorage);
                 this._mockStorage.Add(blob);
```

Comment:
If the MockFileBlob is going to handle the delete, I think for consistency it should handle the Add as well.

Reply:
done